webapp/sources/ldap-inventory/inventory-repository/src/main/scala/com/normation/inventory/ldap/core/InventoryMapper.scala

```diff
@@ -1030,8 +1026,8 @@ class InventoryMapper(
 
       lastLoggedUser     = entry(A_LAST_LOGGED_USER)
       lastLoggedUserTime = entry.getAsGTime(A_LAST_LOGGED_USER_TIME).map(_.dateTime)
-      ev                 = entry.valuesFor(A_EV).toSeq.map(Serialization.read[EnvironmentVariable](_))
-      process            = entry.valuesFor(A_PROCESS).toSeq.map(Serialization.read[Process](_))
+      ev                 = entry.valuesFor(A_EV).toSeq.map(_.fromJson[EnvironmentVariable]).collect { case Right(ev) => ev }
+      process            = entry.valuesFor(A_PROCESS).toSeq.map(_.fromJson[Process]).collect { case Right(p) => p }
```

Comment:
Since the `Serialization.read` method throws, the equivalent would instead be a traverse/ZIO.foreach here instead of failing silently ?

Reply:
the previous case was just a bug I think. 
So what do we want ? Not a full error and an unreadable inventory because a mear process / env var was uncorectly written in base and can't be read back, because it's completely unactionnable by the user. It's even possible that just sending an other inventory doesn't solve the problem by overriding datas, since we read existing one before writting (which means, we may have other problems). Ok, let's write a test on that. 

I can log before ignoring, but I'm pretty sure that log either won't be read ever, or will cause major error like filling disks.

Reply:
Added logs and a test to ensure we can deserialize inventories in presence of incorrect data. 
Alos, did the same for custom properties which were ignored too.